App.js

```diff
@@ -1,41 +1,13 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- * @flow
- */
-
-import React, { Component } from 'react';
-import {
-  Platform,
-  StyleSheet,
-  Text,
-  View
-} from 'react-native';
-import CityList from './src/components/CityList';
-
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' +
-    'Cmd+D or shake for dev menu',
-  android: 'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
-
-export default class App extends Component{
-  render() {
-    return (
-      <View style={styles.container}>
-        <CityList />
-      </View>
-    );
-  }
-}
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-    paddingTop: '20%',
+import { createStackNavigator } from 'react-navigation';
+import CityListScreen from './src/components/CityListScreen';
+import ForecastScreen from './src/components/ForecastScreen';
```

Comment:
It would be good to create `index.js` file in `/src/components` that will export CityListScreen & ForecastScreen. Then you can say `import { CItyListScreen, ForecastScreen } from './src/components';`